gems/pending/spec/disk/modules/azure_blob_disk_spec.rb

```diff
@@ -0,0 +1,152 @@
+require 'ostruct'
+require 'azure-armrest'
+require 'disk/modules/AzureBlobDisk'
+require 'vcr'
+
+describe AzureBlobDisk do
+  before(:all) do
+    @test_env = TestEnvHelper.new(__FILE__)
+    @test_env.vcr_filter
+
+    @client_id     = @test_env[:azure_client_id]
+    @client_key    = @test_env[:azure_client_key]
+    @tenant_id     = @test_env[:azure_tenant_id]
+    @blob_uri      = @test_env[:blob_uri]
+
+    @test_env.ensure_recording_dir_exists
+  end
+
+  before(:each) do |example|
+    Azure::Armrest::ArmrestService.clear_caches
+    example_id = "#{example.example_group.description}-#{example.metadata[:ex_tag]}"
+    cassette_name = @test_env.cassette_for(example_id)
+    VCR.insert_cassette(cassette_name, :decode_compressed_response => true)
+
+    @azure_config = Azure::Armrest::ArmrestService.configure(
+      :client_id  => @client_id,
+      :client_key => @client_key,
+      :tenant_id  => @tenant_id
+    )
+
+    @storage_account_service = Azure::Armrest::StorageAccountService.new(@azure_config)
+  end
+
+  after(:each) do
+    VCR.eject_cassette
+  end
+
+  describe ".new" do
+    it "should raise an error, given a bad blob uri", :ex_tag => 1 do
```

Comment:
Can you reword ex_tag since I can't tell what ex_tag means?  If I need to add examples, I wouldn't know what I need to tag my example with.  It looks like it's used to determine the cassette name but ex_tag doesn't tell me that.